vuepress/docs/v7.0/docs/reference/deployment-structure.md

```diff
@@ -4,154 +4,128 @@ sidebarDepth: 2
 
 # Entando Deployment Structure
 
-This page provides a high level overview of the key Entando GitHub repositories along with a brief description 
-of how those repositories are realized in a running Entando deployment. The descriptions provided here are meant 
-as a guide for identifying opportunities to dig deeper into the architecture and how things are structured 
-rather than a panacea for understanding the architecture.
+This page provides an overview of the key Entando GitHub repositories with a brief description 
+of how these repositories are realized in a running Entando deployment. It also explores the architecture behind Entando and how its functions are structured.
 
 ## entando-operator
-The Entando operator coordinates the installation and configuration of all of the components of an Entando 
-Cluster. The operator can be installed once per Entando Cluster and used to coordinate the plugin lifecycle for 
-multiple Entando applications across many namespaces.
+The Entando Operator coordinates the installation and configuration of all of the components of an Entando 
+cluster. The operator can be installed once per cluster and used to coordinate the plugin lifecycle for multiple Entando applications across many namespaces.
```

Comment:
can be used to coordinate the lifecycle or *is* used to do that? like is that always done or optional? -> "...and is used to manage the plugin lifecycle" (coordinate -> manage)

i think we should capitalize Applications... doesn't make sense to capitalize the singular but not the plural. if Entando directly precedes Application(s) or Bundle(s), etc., these should be capitalized

omit "many"